projects/plugins/jetpack/_inc/client/at-a-glance/search.jsx

```diff
@@ -116,7 +112,7 @@ class DashSearch extends Component {
 			} );
 		}
 
-		if ( ! this.props.isBusinessPlan && ! this.props.hasSearchProduct ) {
+		if ( ! this.props.hasClassicSearch && ! this.props.hasInstantSearch ) {
```

Comment:
@gibrown Does that look right to you? I'm going off of PcNPJE-8W-p2, assuming that the business plan is a WP.com business plan with classic search?

Reply:
I think business plan here can also be the legacy Jetpack Pro plan, but yes I think this is correct.